content/docs/reference-events.md

```diff
@@ -6,13 +6,13 @@ layout: docs
 category: Reference
 ---
 
-This reference guide documents the `SyntheticEvent` wrapper that forms part of React's Event System. See the [Handling Events](/docs/handling-events.html) guide to learn more.
+यह निर्देशिका `SyntheticEvent` आवरण को निर्देशित करता है जो React इवेंट सिस्टम का एक हिस्सा है। अधिक जानने के लिए [Handling Events](/docs/handling-events.html) गाइड देखें।
 
-## Overview {#overview}
+## ओवरव्यू {#overview}
 
-Your event handlers will be passed instances of `SyntheticEvent`, a cross-browser wrapper around the browser's native event. It has the same interface as the browser's native event, including `stopPropagation()` and `preventDefault()`, except the events work identically across all browsers.
+आपके ईवेंट हैंडलर को `SyntheticEvent`, ब्राउज़र के नेटिव ईवेंट के चारों ओर एक क्रॉस-ब्राउज़र आवरण दिया जाएगा। यह ब्राउज़र के नेटिव ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, जब तक कि इवेंट्स, सभी ब्राउज़रों में समान रूप से काम करते हैं।
```

Comment:
the grammar of these sentences is still a bit confusing, revisit once?

Reply:
revisited, and corrected, in case you still feels some confusion, please suggest the edit required.

Reply:
आवरण could be wrapper in hindi instead?

Reply:
second sentence could be reworded as: यह ब्राउज़र के नेटिव ईवेंट के समान है, जिसमें `stopPropagation()` और `preventDefault()` शामिल हैं, par yeh इवेंट्स सभी ब्राउज़रों में समान रूप से काम करते हैं